.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.1.1/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": "./commit.js",
+  "privatePackages": false,
+  "fixed": [["@bifold/*"]],
```

Comment:
This keeps all packages at the same version, regardless if they have or have not been changed